<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DishsByTimeForm</class>
 <widget class="QWidget" name="DishsByTimeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>624</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>51</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="RadialGradientPattern">
       <gradient centralx="0.300000000000000" centraly="-0.400000000000000" focalx="0.300000000000000" focaly="-0.400000000000000" radius="1.300000000000000" type="RadialGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>205</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>51</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>51</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="RadialGradientPattern">
       <gradient centralx="0.300000000000000" centraly="-0.400000000000000" focalx="0.300000000000000" focaly="-0.400000000000000" radius="1.300000000000000" type="RadialGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>205</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="RadialGradientPattern">
       <gradient centralx="0.300000000000000" centraly="-0.400000000000000" focalx="0.300000000000000" focaly="-0.400000000000000" radius="1.300000000000000" type="RadialGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>205</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>51</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="RadialGradientPattern">
       <gradient centralx="0.300000000000000" centraly="-0.400000000000000" focalx="0.300000000000000" focaly="-0.400000000000000" radius="1.300000000000000" type="RadialGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>205</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>51</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>51</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="RadialGradientPattern">
       <gradient centralx="0.300000000000000" centraly="-0.400000000000000" focalx="0.300000000000000" focaly="-0.400000000000000" radius="1.300000000000000" type="RadialGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>205</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="RadialGradientPattern">
       <gradient centralx="0.300000000000000" centraly="-0.400000000000000" focalx="0.300000000000000" focaly="-0.400000000000000" radius="1.300000000000000" type="RadialGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>205</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>51</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="RadialGradientPattern">
       <gradient centralx="0.300000000000000" centraly="-0.400000000000000" focalx="0.300000000000000" focaly="-0.400000000000000" radius="1.300000000000000" type="RadialGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>205</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>51</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>51</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="RadialGradientPattern">
       <gradient centralx="0.300000000000000" centraly="-0.400000000000000" focalx="0.300000000000000" focaly="-0.400000000000000" radius="1.300000000000000" type="RadialGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>205</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="RadialGradientPattern">
       <gradient centralx="0.300000000000000" centraly="-0.400000000000000" focalx="0.300000000000000" focaly="-0.400000000000000" radius="1.300000000000000" type="RadialGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>205</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Трапеза на день</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> 
 QWidget {
color: #333;
border: 2px solid #888888;
font: 12pt &quot;Arial&quot;;
border-radius: 6px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #cdffff);
min-width: 80px;
text-align:left
}
 </string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0" colspan="2">
    <widget class="QPushButton" name="toMainButton">
     <property name="styleSheet">
      <string notr="true"> 
 QPushButton {
color: #333;
border: 2px solid #888888;
font: 30pt &quot;Arial&quot;;
border-radius: 6px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #cdffff);
min-width: 80px;
text-align:left
}
 
QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #bbb);
}
 
 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.3, stop: 0 #fff, stop: 1 #ddd);
}</string>
     </property>
     <property name="text">
      <string>Домой</string>
     </property>
     <property name="icon">
      <iconset resource="images_resources.qrc">
       <normaloff>:/images/img/kfm_home_6893.png</normaloff>:/images/img/kfm_home_6893.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> 
 QLabel {
color: #333;
border: 0px solid #888888;
font: 14pt &quot;Arial&quot;;
border-radius: 6px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #cdffff);
min-width: 80px;
text-align:left
}</string>
     </property>
     <property name="text">
      <string>Перечень блюд:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="dayLabel">
     <property name="styleSheet">
      <string notr="true"> 
 QLabel {
color: #333;
border: 0px solid #888888;
font: 14pt &quot;Arial&quot;;
border-radius: 6px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #cdffff);
min-width: 80px;
text-align:left
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="addButton">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>30</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> 
 QPushButton {
color: #333;
border: 2px solid #888888;
font: 30pt &quot;Arial&quot;;
border-radius: 6px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #cdffff);
min-width: 80px;
text-align:left
}
 
QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #bbb);
}
 
 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.3, stop: 0 #fff, stop: 1 #ddd);
}</string>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
     <property name="icon">
      <iconset resource="images_resources.qrc">
       <normaloff>:/images/img/edit_add_8006.png</normaloff>:/images/img/edit_add_8006.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="trapezaLabel">
     <property name="styleSheet">
      <string notr="true"> 
 QLabel {
color: #333;
border: 0px solid #888888;
font: 14pt &quot;Arial&quot;;
border-radius: 6px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #cdffff);
min-width: 80px;
text-align:left
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QListView" name="dishsByDayListView">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>51</green>
           <blue>51</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="RadialGradientPattern">
          <gradient centralx="0.300000000000000" centraly="-0.400000000000000" focalx="0.300000000000000" focaly="-0.400000000000000" radius="1.300000000000000" type="RadialGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>205</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>51</green>
           <blue>51</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>51</green>
           <blue>51</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="RadialGradientPattern">
          <gradient centralx="0.300000000000000" centraly="-0.400000000000000" focalx="0.300000000000000" focaly="-0.400000000000000" radius="1.300000000000000" type="RadialGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>205</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="RadialGradientPattern">
          <gradient centralx="0.300000000000000" centraly="-0.400000000000000" focalx="0.300000000000000" focaly="-0.400000000000000" radius="1.300000000000000" type="RadialGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>205</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>51</green>
           <blue>51</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="RadialGradientPattern">
          <gradient centralx="0.300000000000000" centraly="-0.400000000000000" focalx="0.300000000000000" focaly="-0.400000000000000" radius="1.300000000000000" type="RadialGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>205</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>51</green>
           <blue>51</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>51</green>
           <blue>51</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="RadialGradientPattern">
          <gradient centralx="0.300000000000000" centraly="-0.400000000000000" focalx="0.300000000000000" focaly="-0.400000000000000" radius="1.300000000000000" type="RadialGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>205</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="RadialGradientPattern">
          <gradient centralx="0.300000000000000" centraly="-0.400000000000000" focalx="0.300000000000000" focaly="-0.400000000000000" radius="1.300000000000000" type="RadialGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>205</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>51</green>
           <blue>51</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="RadialGradientPattern">
          <gradient centralx="0.300000000000000" centraly="-0.400000000000000" focalx="0.300000000000000" focaly="-0.400000000000000" radius="1.300000000000000" type="RadialGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>205</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>51</green>
           <blue>51</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>51</green>
           <blue>51</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="RadialGradientPattern">
          <gradient centralx="0.300000000000000" centraly="-0.400000000000000" focalx="0.300000000000000" focaly="-0.400000000000000" radius="1.300000000000000" type="RadialGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>205</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="RadialGradientPattern">
          <gradient centralx="0.300000000000000" centraly="-0.400000000000000" focalx="0.300000000000000" focaly="-0.400000000000000" radius="1.300000000000000" type="RadialGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>205</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="styleSheet">
      <string notr="true"> 
 QListView {
color: #333;
border: 1px solid #888888;
font: 24pt &quot;Arial&quot;;
border-radius: 1px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #cdffff);
min-width: 80px;
text-align:left
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true"> 
 QLabel {
color: #333;
border: 0px solid #888888;
font: 14pt &quot;Arial&quot;;
border-radius: 6px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #cdffff);
min-width: 80px;
text-align:left
}</string>
     </property>
     <property name="text">
      <string>Название трапезы:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true"> 
 QLabel {
color: #333;
border: 0px solid #888888;
font: 14pt &quot;Arial&quot;;
border-radius: 6px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #cdffff);
min-width: 80px;
text-align:left
}</string>
     </property>
     <property name="text">
      <string>День:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="deleteButton">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>30</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> 
 QPushButton {
color: #333;
border: 2px solid #888888;
font: 30pt &quot;Arial&quot;;
border-radius: 6px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #cdffff);
min-width: 80px;
text-align:left
}
 
QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #bbb);
}
 
 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.3, stop: 0 #fff, stop: 1 #ddd);
}</string>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
     <property name="icon">
      <iconset resource="images_resources.qrc">
       <normaloff>:/images/img/delete_9019.png</normaloff>:/images/img/delete_9019.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPushButton" name="backButton">
     <property name="styleSheet">
      <string notr="true"> 
 QPushButton {
color: #333;
border: 2px solid #888888;
font: 30pt &quot;Arial&quot;;
border-radius: 6px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #cdffff);
min-width: 80px;
text-align:left
}
 
QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.3, stop: 0 #fff, stop: 1 #bbb);
}
 
 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.3, stop: 0 #fff, stop: 1 #ddd);
}</string>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images_resources.qrc"/>
 </resources>
 <connections/>
</ui>
